src/scripts/main.js

```diff
@@ -357,4 +357,50 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+// Функція для розрахунку віку
+const calculateAge = (born, died) => died - born;
+
+// Функція для розрахунку століття
+const calculateCentury = (died) => Math.ceil(died / 100);
+
+// Функція для створення таблиці
+const generateTable = () => {
+  people.forEach(person => {
+    // Розраховуємо вік і століття
+    const age = calculateAge(person.born, person.died);
+    const century = calculateCentury(person.died);
+
+    // Створюємо новий рядок для таблиці
+    const row = document.createElement('tr');
+
+    // Створюємо клітинки для кожного з атрибутів
+    const nameCell = document.createElement('td');
+    nameCell.textContent = person.name;
+
+    const genderCell = document.createElement('td');
+    genderCell.textContent = person.gender;
```

Comment:
The property `gender` is used here, but the correct property name in the `people` array is `sex`. Change `person.gender` to `person.sex` to correctly access the gender information.